jenkins-slaves/jenkins-slave-npm/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'"
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+USER root
+
+ENV NODEJS_VERSION=8 \
+    NPM_CONFIG_PREFIX=$HOME/.npm-global \
+    PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
+    BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+RUN yum remove -y rh-nodejs4; \
+    yum repolist > /dev/null && \
+    INSTALL_PKGS="rh-nodejs8 rh-nodejs8-npm rh-nodejs8-nodejs-nodemon nss_wrapper redhat-lsb libXScrnSaver xdg-utils wget" && \
+    yum install -y --setopt=tsflags=nodocs \
+      --enablerepo=rhel-server-rhscl-7-rpms \
```

Comment:
@etsauer I've seen issues where the repositories are not properly enabled within the container even though it is enabled on the host. It may be due to caching within the container, but further investigation is required. However, I do not believe that it should block this PR as this method is being utilized in other examples. Any changes to the policy should be address at a repo level

Reply:
I agree with @sabre1041 and the (little) investigation I've done backups the "caching" theory. I'll merge this PR as-is as I believe it's ready to go, but we should open an issue to investigate alternative approaches + potentially submit upstream fixes for the cached content within the base images.